website/docs/content/voice-and-tone/index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Voice and Tone
+caption: Applications should be usable and accessible for all users of differing abilities.
+description: Applications should be usable and accessible for all users of differing abilities.
+previewImage: assets/illustrations/foundations/accessibility.jpg
+---
```

Comment:
@majedelass Is it intended for this page to be visible on the site right now?

Reply:
No - page is deleted. My bad!